tests_requirements.txt

```diff
@@ -11,12 +11,22 @@ types-Deprecated
 types-pkg_resources
 types-tabulate
 
-# Notebook tests:
+# Notebooks and documentation:
 ipykernel
+ipython
 jupyter_client
 jupytext
 nbconvert
 nbformat
+nbsphinx
+pandoc
+# Version 0.10.* bad: https://github.com/pydata/pydata-sphinx-theme/issues/952
+pydata-sphinx-theme!=0.10.*
+sphinx
+sphinx-autoapi
+# Version 1.19.3 bad: https://github.com/tox-dev/sphinx-autodoc-typehints/issues/259
+sphinx_autodoc_typehints!=1.19.3
```

Comment:
(FYI 1.19.4 is already out, fixing this)

Reply:
> (FYI 1.19.4 is already out, fixing this)

Yeah, I know. I mean - it's cool they've fixed it so quickly, but I'm also kind of annoyed - I spent hours yesterday tracking down where the crash was coming from, and then it's not even necessary... :-(